api.yaml

```diff
@@ -0,0 +1,194 @@
+openapi: 3.0.0
```

Comment:
Generated by my keyboard :)  Actually I had a mostly complete swagger spec (2.0) and ran it through a converter to change to openapi (3.0) and added the missing stuff.  And yeah I think that is easier than adding all the annotations and auto-generating it since it is a simple API.  But if someone wants to do that sometime more power to them.